### ISSUE TITLE
Link only usage

### ISSUE THREAD

**Issue by knob1** [projectstorm/react-diagrams#904]
I'd like to use the framework just for the linking between nodes. I'd like to draw those links with the grid functionality and place images in between which don't have to be connected to a link. I'd like to create some sort of schematic diagrams / electric circuit diagrams with it. I just explored the demos and found it very useful for this usecase - do you think something like this would be possible?

**Comment by malavshah9** [comment]
@knob1  
Yes, According to what you had described it is possible with custom nodes.

We can make any sort of diagrams using this library.

If you have any other doubts than you can ask here, else you can close this issue.

Thanks!!
